src/webui/api/authcontroller.cpp

```diff
@@ -45,29 +46,41 @@ void AuthController::loginAction()
         return;
     }
 
-    if (isBanned())
+    const QString clientAddr {sessionManager()->clientId()};
+    const QString usernameFromWeb {params()["username"]};
+    const QString passwordFromWeb {params()["password"]};
+
+    if (isBanned()) {
+        LogMsg(tr("WebUI login failure. Reason: IP has been banned, IP: %1, username: %2, password: %3")
```

Comment:
Is it wise to log passwords, even mistyped ones?

Reply:
I think yes, it might reveal something interesting.

Reply:
ok, after some googling I changed my mind, if no one objects, I'll update it later.